### DISCUSSION TITLE
Anyone had any luck getting Telescope repo working on Windows?

### DISCUSSION THREAD

**Discussion by jimofthecorn** [cljoly/telescope-git-repo.nvim#58] [Q&A]
Works great on Linux; on Windows, it displays the interface but doesn't list anything. fd and lolcate are on the PATH. No repo log messages in the quickfix list.

I'm just learning Lua & transitioning to NeoVim-specific plugins, so I'm not sure how to best troubleshoot this.

**Comment by cljoly** [comment]
I don’t run any Windows machine, so I cannot test.

Could you please paste the output of `:checkhealth telescope`? It would help troubleshooting.

**Comment by jimofthecorn** [comment]
After attempt to use "Telescope repo list", "checkhealth telescope" gives me:

```
Telescope: require("Telescope.health").check()

Checking for required plugins ~
- OK plenary installed.
- OK nvim-treesitter installed.

Checking external dependencies ~
- OK rg: found ripgrep 13.0.0
- OK fd: found fd 8.7.0

===== Installed extensions ===== ~

Telescope Extension: `repo` ~
- OK Will use `cat` to preview non-markdown READMEs
- WARNING Install `bat` for a better preview of other files
- OK Will use `cat` to preview markdown READMEs
- WARNING Install `glow` for a better preview of markdown files
- OK locate: found `lolcate`
  Lolcate 0.10.0
- Repos found for `:Telescope repo cached_list`:
  [0m[36mConfig file:, [0m  C:\Users\jim\AppData\Roaming\lolcate\config.toml...
- OK fd: found `fd`
  fd 8.7.0
- ERROR `:Telescope repo list` was unsuccessful. Exit code: 1
  fd --hidden --case-sensitive --absolute-path --exec echo {//} ; ^\.git$
  [fd error]: Command not found: echo
```
Maybe the same as [https://github.com/cljoly/telescope-repo.nvim/issues/43](https://github.com/cljoly/telescope-repo.nvim/issues/43)?

**Reply by cljoly** [reply]
Yes, I you might have the same issue, [this comment](https://github.com/cljoly/telescope-repo.nvim/issues/43#issuecomment-1356355777) might solve your problem. If you want, you can change the code of the plugin to test. If you don’t see the change to make, I’ll make it this weekend for you.
